src/vivarium_ciff_sam/data/loader.py

```diff
@@ -86,7 +78,8 @@ def load_theoretical_minimum_risk_life_expectancy(key: str, location: str) -> pd
 def load_standard_data(key: str, location: str) -> pd.DataFrame:
     key = EntityKey(key)
     entity = get_entity(key)
-    return interface.get_measure(entity, key.measure, location)
+    data = interface.get_measure(entity, key.measure, location).droplevel('location')
```

Comment:
this fixes the data so that it matches the structure that the framework now expects. Long term we should figure out if we want to have multiple locations live in the artifact, or if we only want one. Either way, this is not a long term solution, as the should either move to the interface or somewhere else.

Reply:
One approach:
```
from vivarium_inputs import interface

def _get_measure(*args, **kwargs):
    return interface.get_measure(*args, **kwargs).droplevel('location')
    
interface.get_measure = _get_measure

Reply:
This is mostly incase you need to use get_measure in multiple places.